docs/meetups/index.rst

```diff
@@ -1,101 +1,84 @@
 Meetups
 =======
```

Comment:
@barbaricyawps can I get you to review the new edits I made to the main page? Thanks!

Reply:
I had a thought re: your call to let us know if they want to re-activate (can't remember the line number). Sometimes people are eager to be an organizer but they don't have a sense of whether there are enough interested members in their area to sustain an in-person meetup.

I wonder if a good first step would be to encourage people to put out an informal call on Slack to see how many documentarians are in a particular area and whether they would like to meet a) online, b) in-person, c) hybrid.  I feel like a suggestion of 10-15 people who seem interested might be a good ballpark, but it really depends on how enthusiastic they are about meeting. You could do it with less if, say, 4-6 people were very committed to attending and the rest were lukewarm.

Reply:
Great point @barbaricyawps, I actually have another branch open where I am updating content in the meetup organizer guide. This would be a great edit/update there.